source/educators/concepts/proctored_exams/proctored_exams_overview.rst

```diff
@@ -1,8 +1,8 @@
 .. _ProctoredExams_Overview:
 
-############################
+#######################
 Proctored Exam Overview
```

Comment:
This file has a lot of references to edX that should be changed or removed. I actually wonder if any of this is applicable to Open edX (as opposed to just edx.org). Before editing please check in with @jswope00

Reply:
Please leave this doc in, and fix all the edX references in it.

Reply:
I'm going to mark this PR passed and merge it since this is an RST cleanup PR. But @jesgreco a future PR should resolve the edX.org references in this file.